middleware/domain/source/manager/handle.cpp

```diff
@@ -601,10 +601,19 @@ namespace casual
                            else
                               log::line( log::category::information, "process exited, alias: ", alias, ", details: ", message.state);
 
+                           auto scale = [ &state]( auto& executable)
+                           {
+                              auto spawnable = executable->spawnable();
+                              handle::scale::instances( state, *executable);
+                              if( spawnable)
+                                 executable->restarts++;
+                           };
```

Comment:
* What if scale::instances fails? 
* What if there are more than one _spawnable_? 

It's really unfortunate that "we" picked _restarts_... It gets way more complicated then one would like...

Should we rethink this?

Reply:
What if we keep track of deaths/exits? And then calculated the restarts for the cli? No that wont work...

Reply:
I've been looking for a less "complex" way... The only thing that seems reasonable is to increment `restarts` in  `state::Server::remove( common::strong::process::id pid)` and  `state::Executable::remove( strong::process::id pid)`.  That's when we go from `running` to  `scale_out` via _restart_ .  The meaning of _restarts_ becomes more like _scheduled_restarts_, but to me it feels more sane. 
What do you think @ahhud